assets/stylesheets/_components.scss

```diff
@@ -66,7 +66,7 @@
 //@import 'components/mobile-back-to-sidebar/style';
 @import 'components/notice/style';
 //@import 'components/olark-chatbox/style';
-//@import 'components/payment-logo/style';
+@import 'components/payment-logo/style';
```

Comment:
Should we use this opportunity to remove the commented-out components? We can import the Calypso component style manually when we start using a new one.